angular-ngrx-scss/src/app/auth/auth.component.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
- [ ] Please delete the test file if there's nothing worth testing in the component.